core/src/avm2/globals/flash/display/bitmap.rs

```diff
@@ -22,22 +23,13 @@ pub fn init<'gc>(
         activation.super_init(this, &[])?;
 
         let bitmap_data = args
-            .get(0)
-            .cloned()
-            .unwrap_or(Value::Null)
-            .as_object()
-            .and_then(|bd| bd.as_bitmap_data());
+            .try_get_object(activation, 0)
+            .and_then(|o| o.as_bitmap_data());
         //TODO: Pixel snapping is not supported
         let _pixel_snapping = args
-            .get(1)
-            .cloned()
-            .unwrap_or_else(|| "auto".into())
-            .coerce_to_string(activation)?;
-        let smoothing = args
-            .get(2)
-            .cloned()
-            .unwrap_or_else(|| false.into())
-            .coerce_to_boolean();
+            .try_get_string(activation, 1)?
+            .unwrap_or_else(|| "auto".into());
```

Comment:
I think pixel snapping is automatically set to "auto" by the following AS code in Bitmap.as:
`public function Bitmap(bitmapData:BitmapData = null, pixelSnapping:String = "auto", smoothing:Boolean = false) {`

Reply:
Removed it